app/controllers/application_controller.rb

```diff
@@ -25,7 +25,7 @@ def current_permission
 
   def authorize!
     redirect_to root_url,
-      warning: "You are an unauthorized boat owner!!" unless authorized?
+      warning: "Bold move Cotton but you are an unauthorized boat owner!!" unless authorized?
```

Comment:
Line is too long. [93/80]<br>Align the parameters of a method call if they span more than one line.